### ISSUE TITLE
ninja: build stopped: subcommand failed (FAILED: icall-table.json)

### ISSUE THREAD

**Issue by nor0x** [unoplatform/Uno.Wasm.Bootstrap#647]

**Comment by jeromelaban** [comment]
Also, can you provide the full binlog file? It may help to troubleshoot the reason for the cross compiler to kick in.

**Comment by nor0x** [comment]
thanks for the quick response!
is there a different way to disable aot? I already have

`<WasmShellMonoRuntimeExecutionMode>Interpreter</WasmShellMonoRuntimeExecutionMode>`

in my .csproj file. I will prepare the binlog file

**Comment by jeromelaban** [comment]
No, setting this mode should be enough. This would mean that for some reason, the AOT compiler is still incorrectly being used.
